Wallet, Explorer and Faucet Services (#43)
nemtech/symbol-bootstrap: CHANGELOG.md

---|---|---
Symbol Bootstrap | v0.3.0 | [symbol-bootstrap](https://www.npmjs.com/package/symbol-bootstrap)

> - **New Service:** `Walelt`. Bootstrap private network starts a Wallet service in [http://localhost:80/](http://localhost:90/) when using `--assembly full`. . **Warning:** This wallet service is for demonstration proposes.

Reviewer: ```suggestion
- **New Service:** `Wallet`. Bootstrap private network starts a Wallet service in [http://localhost:80/](http://localhost:90/) when using `--assembly full`. . **Warning:** This wallet service is for demonstration purposes only.
```

Port 80 or 90!?!?!?

> - **New Service:** `Wallet`. Bootstrap private network starts a Wallet service in [http://localhost:80/](http://localhost:80/) when using `--assembly full`. . **Warning:** This wallet service is for demonstration proposes only.

Reviewer: ```suggestion
- **New Service:** `Wallet`. Bootstrap private network starts a Wallet service in [http://localhost:80/](http://localhost:80/) when using `--assembly full`. . **Warning:** This wallet service is for demonstration purposes only.
```

Reviewer: Be careful! You missed one! :D

> - **New Service:** `Explorer`. Bootstrap private network starts a Faucet service in [http://localhost:90/](http://localhost:80/) when using `--assembly full`. 

Reviewer: Port 90 or 80?

Author: Fixed, 80 is the wallet 90 the explorer

> - **New Service:** `Faucet`. Bootstrap private network starts an Explorer service in [http://localhost:100/](http://localhost:100/) when using `--assembly full`. 

Reviewer: ```suggestion
- **New Service:** `Faucet`. Bootstrap private network starts a Faucet service in [http://localhost:100/](http://localhost:100/) when using `--assembly full`. 
```

Author: Fixed
